test/WebJobs.Script.Tests/Diagnostics/LinuxAppServiceEventGeneratorTests.cs

```diff
@@ -175,31 +175,31 @@ public void ParseAzureMonitoringEvents(LogLevel level, string resourceId, string
                 p => Assert.True(DateTime.TryParse(p, out DateTime dt)));
         }
 
-        [Theory]
-        [MemberData(nameof(LinuxEventGeneratorTestData.GetFunctionExecutionEvents), MemberType = typeof(LinuxEventGeneratorTestData))]
-        public void ParseFunctionExecutionEvents(string executionId, string siteName, int concurrency, string functionName, string invocationId,
-            string executionStage, long executionTimeSpan, bool success)
-        {
-            _generator.LogFunctionExecutionEvent(executionId, siteName, concurrency, functionName, invocationId, executionStage, executionTimeSpan, success);
-            string evt = _loggers[LinuxEventGenerator.FunctionsExecutionEventsCategory].Events.Single();
-
-            Regex regex = new Regex(LinuxAppServiceEventGenerator.ExecutionEventRegex);
-            var match = regex.Match(evt);
-
-            Assert.True(match.Success);
-            Assert.Equal(10, match.Groups.Count);
-
-            var groupMatches = match.Groups.Cast<Group>().Select(p => p.Value).Skip(1).ToArray();
-            Assert.Collection(groupMatches,
-                p => Assert.Equal(executionId, p),
-                p => Assert.Equal(siteName, p),
-                p => Assert.Equal(concurrency.ToString(), p),
-                p => Assert.Equal(functionName, p),
-                p => Assert.Equal(invocationId, p),
-                p => Assert.Equal(executionStage, p),
-                p => Assert.Equal(executionTimeSpan.ToString(), p),
-                p => Assert.True(Convert.ToBoolean(p)),
-                p => Assert.True(DateTime.TryParse(p, out DateTime dt)));
-        }
+        // [Theory]
```

Comment:
Please do not leave commented out code. If this is something you want to keep around sot it is reinstated, please create an issue for tracking and reference the changes.

Reply:
Got tests working thank you for the nudge